docs/api/web-contents.md

```diff
@@ -403,6 +403,12 @@ a meta tag:
 <meta name='theme-color' content='#ff0000'>
 ```
 
+Returns:
+
+* `event` Event
+* `color` string or `null`
```

Comment:
I know you just changed this but the correct notation for our documentation parser is:

```markdown
(String | null)
```

Note the brackets and no backticks